src/Utilities/Compiler/WellKnownTypeNames.cs

```diff
@@ -53,13 +53,13 @@ internal static class WellKnownTypeNames
         public const string MicrosoftCodeAnalysisDiagnosticsSyntaxNodeAnalysisContext = "Microsoft.CodeAnalysis.Diagnostics.SyntaxNodeAnalysisContext";
         public const string MicrosoftCodeAnalysisDiagnosticsSyntaxTreeAnalysisContext = "Microsoft.CodeAnalysis.Diagnostics.SyntaxTreeAnalysisContext";
         public const string MicrosoftCodeAnalysisHostMefMefConstruction = "Microsoft.CodeAnalysis.Host.Mef.MefConstruction";
-        public const string MicrosoftCodeAnalysisLocalizableString = "Microsoft.CodeAnalysis.LocalizableString";
```

Comment:
@mavasani Do you think the code style repo would accept a suggestion for an analyzer that sorts members by name? Shall we do something specific to `WellKnownTypeNames`?

Reply:
Can we enable FxCop on this repo?